ManageWikiSettings.php

```diff
@@ -363,6 +362,51 @@
 		'section' => 'edit',
 		'help' => 'Actions that can be restricted.',
 	],
+	'wgCommentStreamsEnableTalk' => [
+		'name' => 'CommentStreams Enable Talk',
+		'from' => 'commentstreams',
+		'restricted' => false,
+		'type' => 'check',
+		'overridedefault' => false,
+		'section' => 'edit',
+		'help' => 'Enable in talk namespaces',
+	],
+	'wgCommentStreamsNewestStreamsOnTop' => [
+		'name' => 'Order of comments',
```

Comment:
Oh, i missed to add that you should add CommentStream here and also below too, sorry that i missed that.